### ISSUE TITLE
Report progress during extentions installation

### ISSUE THREAD

**Issue by serverhorror** [hpcugent/easybuild-framework#1507]
It would be nice to have some form of feedback during the `== taking care of extensions...` phase. Some easyconfigs spend the majority of time in there so any indication of progress would be nice.

Something like this:

```
...
== testing...
== installing...
== taking care of extensions...
== * Working on extension a-1.23
== * Working on extension a-1.23
== * Working on extension a-1.23
== * Working on extension a-1.23
== postprocessing...
...
```

**Comment by boegel** [comment]
@serverhorror: up for taking a stab at it yourself? it's fairly trivial

see https://github.com/hpcugent/easybuild-framework/blob/master/easybuild/framework/easyblock.py#L1550, to print, use the `print_msg` function

**Comment by serverhorror** [comment]
Should this be reported always or maybe just on `eb --force --skip ...` or maybe even the other way around (only when building initially)?

I've been spamming myself with messages and I'm not convinced anymore that it is a good idea to report that level of detail unconditionally

**Comment by boegel** [comment]
@serverhorror: there are several options

* print the name/version for extensions being installed one-by-one but all on the same line (to avoid scrolling)
* print a N/M overview of how many extensions are left to be installed, and refresh it (one the same line)
* just print a '`.`' or `'x'` for every extension

**Comment by serverhorror** [comment]
Hi,

I like the second option.Is adding a new dependency OK? I'll look into
progress bar libs then.

/Martin

On Tue, Jan 26, 2016 at 11:37 AM, Kenneth Hoste <notifications@github.com>
wrote:

> @serverhorror <https://github.com/serverhorror>: there are several options
>
>    - print the name/version for extensions being installed one-by-one but
>    all on the same line (to avoid scrolling)
>    - print a N/M overview of how many extensions are left to be
>    installed, and refresh it (one the same line)
>    - just print a '.' or 'x' for every extension
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hpcugent/easybuild-framework/issues/1507#issuecomment-174949872>
> .
>



-- 
http://www.xing.com/profile/Martin_Marcher
http://www.linkedin.com/in/martinmarcher
Mobil: +43 / 660 / 62 45 103
UID: ATU68801424

**Comment by boegel** [comment]
fixed in #1886, thanks @serverhorror!
